stubs/PyMySQL/pymysql/_auth.pyi

```diff
@@ -0,0 +1,21 @@
+from _typeshed import Incomplete
+from typing import Final
+
+DEBUG: Final[bool]
+SCRAMBLE_LENGTH: Final[int]
+sha1_new: Incomplete
+
+def scramble_native_password(password: str | bytes | None, message: str | bytes | None) -> bytes: ...
+def _my_crypt(message1: str | bytes, message2: str | bytes) -> bytes: ...
```

Comment:
Let's skip this private function (and the other ones below) until and unless someone actually needs them.